packages/donate-button-next/src/components/widget/index.tsx

```diff
@@ -303,15 +304,22 @@ const Widget = ({options, hide}: WidgetProps) => {
 			return;
 		}
 
-		const url = constructEveryUrl({
+		let options = {
```

Comment:
```suggestion
		const options = {
```

It seems like `options` can be a `const` since the variable never reassigned.
Object.assign doesn't need let, it interacts with the object, doesn't reassign the variable.

Reply:
Yup, this is just a personal preference to indicate that the object value *might* be mutated!